src/Synapse/Synapse/Common/HelpMessages.cs

```diff
@@ -265,10 +265,16 @@ public static class HelpMessages
 
         public const string RoleDefinitionId = "Id of the Role that is assigned to the principal.";
 
+        public const string RoleAssignmentItemType = "Item type of the role assignment:'bigDataPools', 'integrationRuntimes', 'linkedServices', 'credentials'.";
+
+        public const string RoleAssignmentItem = "Item of the Role that is assigned to the principal.";
+
         public const string RoleDefinitionName = "Name of the Role that is assigned to the principal.";
 
         public const string PrincipalId = "The Azure AD ObjectId of the User, Group or Service Principal.";
 
+        public const string Scope = "The Scope of the user.";
```

Comment:
>The Scope of the user. [](start = 37, length = 22)

Where do you get this description? Can we improve it? For example, from Synapse studio, I see the description is "The part of the workspace to which you want to grant access."

Reply:
Ping

---
In reply to: [599501756](https://github.com/Azure/azure-powershell/pull/14172#discussion_r599501756) [](ancestors = 599501756)